### PR TITLE
set real build-start/sheduled-time as PointTime in JenkinsBasePointGenerator

### PR DESCRIPTION
**Pull request by nubody**
Overwrite buildPoint()-Method in JenkinsBasePointGenerator to set real build-start/sheduled-time as PointTime instead of time when Point is created (when BuildJob is done or in PostProcess).

### CODE REVIEW

**Comment by asimell** [comment]
@nubody Can you redirect this pull request to the development branch instead of master?

**Comment by nubody** [comment]
Done

**Comment by nubody** [comment]
@asimell Hi Aleksi,
is this change, or rather the approach, ok for you?

We noticed, that the 'time' of a Point is set, when the Point is finally build/created, except, the 'time' was set before (what now is done in the JenkinsBasePointGenerator; as it was already implemented in the PerfPublisherPointGenerator too).
So until now the Timestamp is always the time, when the jenkins-job is (nearly) done and not, when the job was scheduled / executed.

**Comment by asimell** [comment]
@nubody There is nothing wrong with the approach. 

However, as I commented on https://github.com/jenkinsci/influxdb-plugin/pull/42 earlier this week, the problem is that at least `RobotFrameworkPointGenerator` creates multiple Points in `tag_point`, `testcase_point`, and `suite_result`. InfluxDB uses time as a primary key and simply replaces a stored value if the time is the same. That's why RF results should still have the time as it is now with tiny sleeps to ensure a different timestamp for each RF point.

In other words, the plugin requires certain measurements to have different timestamps. There needs to be a way for those few measurements to have a different timestamp even if all the other measurements had the same.

**Comment by nubody** [comment]
Closed, since the functionality is covered by #42
